include/quicr/quicr_client_session.h

```diff
@@ -0,0 +1,164 @@
+/*
+ *  quicr_client_session.h
+ *
+ *  Copyright (C) 2023
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ *
+ *  Description:
+ *      This is an interface specification for the session layer sitting at the
+ *      next level of the client library.
```

Comment:
may be clarify what is the next level here by drawing an ascii layer cake

Reply:
I'll add a topology comment.